src/Build/BuildCheck/Acquisition/BuildCheckAcquisitionModule.cs

```diff
@@ -52,7 +52,8 @@ public List<BuildAnalyzerFactory> CreateBuildAnalyzerFactories(AnalyzerAcquisiti
 
             if (availableTypes.Count != analyzerTypes.Count)
             {
-                availableTypes.Except(analyzerTypes).ToList().ForEach(t => _loggingService.LogComment(buildEventContext, MessageImportance.Normal, "CustomAnalyzerBaseTypeNotAssignable", t.Name, t.Assembly));
+                availableTypes.Except(analyzerTypes).ToList()
+                    .ForEach(t => _loggingService.LogComment(buildEventContext, MessageImportance.Normal, "CustomAnalyzerBaseTypeNotAssignable", t.Name, t.Assembly));
```

Comment:
Are we able to get the package name as well?

Reply:
we operate with dlls directly. I can have a location reported.

Reply:
it's already there
![image](https://github.com/dotnet/msbuild/assets/95473390/b35d35db-25b6-47e4-b0e7-380fb20be161)